sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -579,11 +580,50 @@ class Dataset[T] private[sql](
    */
   def takeAsList(num: Int): java.util.List[T] = java.util.Arrays.asList(take(num) : _*)
 
+
+  /* ******* *
+   *  Cache  *
+   * ******* */
+
+  /**
+    * @since 1.6.0
+    */
```

Comment:
@rxin I think thats only because we explicitly exclude execution from scaladoc.  Maybe we should move queryable?  or don't exclude that class.  I don't want to duplicate a ton of docs.